doc/scapy/build_dissect.rst

```diff
@@ -868,7 +868,10 @@ Legend:
     LEShortField
     XShortField
     
-    X3BytesField        # three bytes (in hexad 
+    X3BytesField        # three bytes as hex
+    LEX3BytesField      # three bytes as decimal
```

Comment:
Hmm, X is for hex :/

Reply:
I agree, it's odd. The old code had `(in hexad` in it, so I thought I'd carry it through. Happy to remove all those comments, as they're not really needed IMO.